wordpress-mobile/WordPress-Android
# Add 'read source post' link to comment

Fixes #7852 
Adds a _Read the source post_ button to comment detail.


![screenshot_1528183297](https://user-images.githubusercontent.com/1079756/40961107-e70ab9ce-68a1-11e8-95d1-f5ca71222824.png)

To test:
* Go to comment
* You see a 'Read the source post link'
* Click the link
* Source post is opened
